nerfstudio/cameras/lidars.py

```diff
@@ -22,6 +22,7 @@
 
 import numpy as np
 import torch
+#import torch._dynamo
```

Comment:
Please remove unused code :)

Reply:
Will do, this was required with specific version of torch I ended-up having in a environment... 
Probably a better environment management could have prevented this being required. 
If it can help in the future, the following code snipped could replace the two commented lines of codes. 

```
        import torch
        if torch.__version__.split("+")[0] >= "2.1.0":
            import torch._dynamo
            torch._dynamo.config.suppress_errors = True
```

Reply:
Done